### PR TITLE
3.x

### PR DESCRIPTION
**Pull request by stevelitras**
Deprecation of logstream-master chart
Introduction of logstream-leader chart
Introduction of loadBalancerIP capability in both leader and workgroup charts

Addresses #31 and #37

### CODE REVIEW

File: helm-chart-sources/logstream-master/README.md [hunk 1]
```diff
@@ -4,9 +4,9 @@
 
 This Chart deploys a Cribl LogStream master server.
 
-# DISCLAIMER
+# Deprecation Notice
+As of this version, 2.9.9, this chart has been deprecated, in favor of the [logstream-leader](../logstream-leader/) chart. You *can* still run this chart with newer versions of the LogStream product (see the section on [Installing a specific version of LogStream](#versionInstall)
```
Comment (mQkatz) [inline helm-chart-sources/logstream-master/README.md:8]: I'd pull the comma after "deprecated" and use your credit to append a period at the end of the paragraph. 😉

File: helm-chart-sources/logstream-master/README.md [hunk 2]
```diff
@@ -77,6 +77,18 @@ This section covers the most likely values to override. To see the full scope of
 
   `helm install logstream-master cribl/logstream-master --set config.scName='ebs-sc'`
   
+# <a name="versionInstall"></a>Installing using a specific version of LogStream
+
+LogStream container images use semantic versioning in the tagging scheme; every "version" tag consists of `<major>.<minor>.<patch>`. For example, if you want the version 3.0.2 of LogStream, you'd set your value for `criblImage.tag` to `3.0.2`, in the install command:
```
Comment (mQkatz) [inline helm-chart-sources/logstream-master/README.md:82]: Suggest streamlining 2nd sentence to just:
>For example, if you want LogStream version 3.0.2, you'd set your `criblImage.tag` value to `3.0.2` in the install command:

File: helm-chart-sources/logstream-master/README.md [hunk 3]
```diff
@@ -77,6 +77,18 @@ This section covers the most likely values to override. To see the full scope of
...
+
+  `helm install logstream-master cribl/logstream-master --set criblImage.tag="3.0.2"`
+  
+and if you're upgrading a pre-existing release, just use the upgrade command:
```
Comment (mQkatz) [inline helm-chart-sources/logstream-master/README.md:86]: "and" s/b either "And" or "...and"

**Review by mQkatz — APPROVED** [review]
Suggested 3 small wording/formatting changes. Otherwise LGTM.

**Review by coccyx — APPROVED** [review]

**Pull request merged**